stable/rabbitmq/README.md

```diff
@@ -78,10 +78,19 @@ The following table lists the configurable parameters of the RabbitMQ chart and
 | `rabbitmq.configuration`             | Required cluster configuration                   | See values.yaml                                         |
 | `rabbitmq.extraConfiguration`        | Extra configuration to add to rabbitmq.conf      | See values.yaml                                         |
 | `rabbitmq.advancedConfiguration`     | Extra configuration (in classic format) to add to advanced.config    | See values.yaml                                         |
+| `rabbitmq.tls.enabled`                  | Enable ssl support to rabbitmq |   | `false`                     |
```

Comment:
`Enable ssl support...` -> `Enable TLS support...`

Reply:
changed to tls